app/assets/stylesheets/_related.scss

```diff
@@ -26,20 +26,16 @@
   color: $text-colour;
   margin-top: 0.3em;
   margin-bottom: 0.5em;
-  position: relative
-}
+  position: relative;
```

Comment:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3069/files#r344819521)